Adyen/UI/Presentation/ModalViewController.swift

```diff
@@ -44,10 +43,7 @@ internal final class ModalViewController: UIViewController {
     }
     
     /// :nodoc:
-    internal var isRoot: Bool = false
-    
-    /// :nodoc:
-    internal var cancelButtonHandler: ((Bool) -> Void)?
+    public var isRoot: Bool = false
```

Comment:
why is this public? this is internal logic

Reply:
true..
could be even private